src/engine/keycontrol.cpp

```diff
@@ -11,9 +11,10 @@
 
 //static const double kLockOriginalKey = 0;
 static const double kLockCurrentKey = 1;
+static const double kUnlockKeyToRate = 1;
 
 KeyControl::KeyControl(QString group,
-                       UserSettingsPointer pConfig)
+						UserSettingsPointer pConfig)
         : EngineControl(group, pConfig) {
```

Comment:
Hm...in git diff the indendation offset was shown the other way around. Will fix.